### DISCUSSION TITLE
Subfamily?

### DISCUSSION THREAD

**Discussion by tripflex** [coinfoundry/miningcore#1552] [Q&A]
Can you provide any details on subfamily and the intentions behind it? It doesn't seem to be used at all so don't have any examples to go off of, is this meant for not having to create a full new family and only override specific family handling? I'm looking at adding another coin that uses a few different RPC calls and I think this may be the best route to go, just wondering if there's some magic handling for it that I'm not seeing or understanding fully?

Thanks for all your hard work on this project!!

**Comment by oliverw** [comment]
It was intended for that purpose but never saw any use because ... well I don't really know why 😁
